android/src/main/java/com/google/android/cameraview/Camera2.java

```diff
@@ -727,19 +770,16 @@ private boolean chooseCameraIdByFacing() {
                 int internalFacing = INTERNAL_FACINGS.get(mFacing);
                 final String[] ids = mCameraManager.getCameraIdList();
                 if (ids.length == 0) { // No camera
-                    throw new RuntimeException("No camera available.");
+                    Log.e(TAG, "No cameras available.");
```

Comment:
What do you think about returning something to the JS? Some people don't know how to see the `Logcat` provided  by android. Removing the `throw`(that probably show which error triggered that) can lead some confusion.